identity/src/test/groovy/life/qbic/identity/domain/model/token/PersonalAccessTokenEncoderTest.groovy

```diff
@@ -0,0 +1,116 @@
+package life.qbic.identity.domain.model.token
+
+import spock.lang.Specification
+
+class PersonalAccessTokenEncoderTest extends Specification {
```

Comment:
NIT: Shouldn't the test be called PersonalAccessTokenEncoderSpec to adhere to our testing naming scheme?

Reply:
Yes